client/js/components/user/CustomerSettings/CustomerSettingsContact.vue

```diff
@@ -0,0 +1,241 @@
+            data-dp-validate-error="error.customer.contact.title"
+            :placeholder="Translator.trans('customer.contact.title')"
+            required
+            type="text" />
+          <dp-input
+            id="phoneNumber"
+            v-model="customerContact.phoneNumber"
+            autocomplete="tel"
+            class="u-mb-0_75"
+            data-cy="phoneNumber"
+            data-dp-validate-error="error.customer.contact.phone_or_email"
+            pattern="^(\+?)(-| |[0-9]|\(|\))*$"
+            :placeholder="Translator.trans('customer.contact.phone_number')"
+            :required="phoneIsRequired"
+            type="tel"
+            @input="input => setRequiredEmail(input)" />
+          <dp-input
+            id="emailAddress"
+            v-model="customerContact.eMailAddress"
+            autocomplete="email"
+            class="u-mb-0_75"
+            data-cy="emailAddress"
+            data-dp-validate-error="error.customer.contact.phone_or_email"
+            :placeholder="Translator.trans('email.address')"
+            :required="emailIsRequired"
+            type="email"
+            @input="input => setRequiredPhone(input)" />
+          <dp-editor
+            id="supportText"
+            class="u-mb-0_75"
+            v-model="customerContact.text"
+            hidden-input="supportText"
+            :toolbar-items="{
+              fullscreenButton: true,
+              headings: [2,3,4],
+              linkButton: true
+            }" />
+          <dp-checkbox
+            id="contactVisible"
+            v-model="customerContact.visible"
+            data-cy="contactVisible"
+            :label="{
+              text: Translator.trans('customer.contact.visible')
+            }" />
+        </div>
+      </template>
+    </dp-editable-list>
+  </div>
+</template>
+
+<script>
+import { DpCheckbox, DpEditableList, DpEditor, DpInput, dpValidateMixin } from '@demos-europe/demosplan-ui'
+import { mapActions, mapMutations, mapState } from 'vuex'
+
+export default {
+  name: 'CustomerSettingsContact',
+
+  components: {
+    DpCheckbox,
+    DpEditableList,
+    DpEditor,
+    DpInput
+  },
+
+  mixins: [dpValidateMixin],
+
+  data () {
+    return {
+      emailIsRequired: true,
+      phoneIsRequired: true,
+      showContactForm: false,
+      translationKeys: {
+        new: Translator.trans('customer.contact.new'),
+        add: Translator.trans('customer.contact.add'),
+        abort: Translator.trans('abort'),
+        update: Translator.trans('customer.contact.update'),
+        noEntries: Translator.trans('customer.contact.no'),
+        delete: Translator.trans('customer.contact.delete')
+      }
+    }
+  },
+
+  computed: {
+    ...mapState('customerContact', {
+      contacts: 'items'
+    }),
+
+    customerContact () {
+      return {
+        title: '',
+        phoneNumber: '',
+        eMailAddress: '',
+        text: '',
+        visible: false
+      }
+    }
```

Comment:
this looks somehow strange. what gets computed here. Like so, its always the Object with the "default"-Values.
If You use a computed for v-modeling, You have to use getter and setter afaik.